README.md

```diff
@@ -93,19 +95,170 @@ Watson Health Cognitive Services | [Go SDK](https://github.com/IBM/whcs-go-sdk)
+<dependency>
+    <groupId>com.ibm.cloud</groupId>
+    <artifactId>example-service</artifactId>
+    <version>1.0.0</version>
+</dependency>
+```
+
+For gradle, the dependency would look like this:
+
+```
+compile 'com.ibm.cloud:example-service:1.0.0'
+```
+
+Be sure to consult the service table for the Java SDK client library you are using
+to obtain the correct artifact coordinates for the service(s) you will use in your application.
+
+After defining an appropriate dependency within your build script, you would then import the
+classes needed by your application.   For example, in order to construct an instance of the
+"Example Service" service client in your application, you would need to import the
+java class that implements that service, like this:
+
+```java
+import com.ibm.cloud.mysdk.example_service.v1.ExampleService;
+```
+
+Your application will likely use a collection of various classes provided by the
+service's jar in addition to the primary class for that service.   Be sure to include
+import statements in your application for each class that you need to use from the
+service's jar.
+
+Note that some Java SDK client libraries provide a collection of javadocs for the various
+classes provided by that client library (look for a link on the Java SDK project's main
+github.com page).  If present, these javadocs can be consulted to explore the
+set of available classes and determine which classes might be needed by your application.
+</details>
+<details><summary>Node.js</summary>
+For Node.js SDK client libraries, each IBM Cloud service exists within its own folder inside the
+package that represents the SDK project as a whole.
+To use the mythical "Example Service" service client in your Node.js application, add an import
+to your application like this:
+
+```js
+const ExampleServiceV1 = require('mysdk/example-service/v1');
+```
+
+You would also need to add a dependency for the `mysdk` package within your application project's
+`package.json` file as well.
+
+Note: this example assumes that the "Example Service" service client exists in the
+`example-service` folder contained within the `mysdk` package.
+
+Be sure to consult the service table for the Node.js SDK client library you are using
+to obtain the correct import path for the service(s) you will use in your application.
+</details>
+<details><summary>Python</summary>
+For Python SDK client libraries, each IBM Cloud service exists in its own module inside the
+package that represents the SDK project as a whole.
+To use the mythical "Example Service" service client in your Python application,
+add an import like this:
+
+```python
+from mysdk import ExampleServiceV1
+```
+
+This imports the `ExampleServiceV1` class, which is the primary class associated with
+the "Example Service" service client.
+If your application needs to use additional service-related definitions (besides the
+service class), use an import like this:
+
+```python
+from mysdk.example_service_v1 import *
```

Comment:
We should suggest importing named definitions, like 
```python
from mysdk.example_service_v1 import Resource, Resources
```

since [`import *` is discouraged in PEP8](https://www.python.org/dev/peps/pep-0008/#imports), which we use as our Python coding style guide.

Reply:
updated